packages/config/src/android/NavigationBar.ts

```diff
@@ -26,9 +26,6 @@ export async function setNavigationBarConfig(config: ExpoConfig, projectDirector
 
   const stylesPath = await getProjectStylesXMLPathAsync(projectDirectory);
   const colorsPath = await getProjectColorsXMLPathAsync(projectDirectory);
-  if (!colorsPath || !stylesPath) {
```

Comment:
if `getProjectColorsXMLPathAsync` throws, maybe the developer won't get the most useful error message here. better than no error message when we silently fail like before by just returning false